### PR TITLE
feat: add x and threads icons

### PR DESCRIPTION
**Pull request by braddialpad**
## Description
Adds the X and threads icon

## Pull Request Checklist

 - [x] Ask the contributors if a similar effort is already in process or has been solved.
 - [x] Review the [contribution guidelines](https://github.com/dialpad/dialtone-icons/blob/staging/.github/CONTRIBUTING.md).
 - [x] Use `staging` as your pull request's base branch. (All PRs using `production` as its base branch will be declined).
 - [x] Ensure all `gulp` scripts successfully compile.

### CODE REVIEW

File: src/icons.json [hunk 1]
```diff
@@ -240,6 +241,7 @@
       "visa": [],
       "we-chat": [],
       "whatsapp": [],
+      "x-brand": [],
```
Comment (francisrupert) [inline src/icons.json]: Add `twitter` as a keyword for `x-brand`.

File: src/icons.json [hunk 2]
```diff
@@ -232,6 +232,7 @@
       "slack": [],
       "snapchat": [],
       "telegram": [],
+      "threads": [],
```
Comment (francisrupert) [inline src/icons.json]: Add `meta` and `facebook` as keywords for `threads`.

**Review by francisrupert — APPROVED** [review]
Approve after requested changes: keywords.

See how keywords are done on this `icons.json`.

**Comment by braddialpad** [comment]
Great idea. Any other keywords you want to throw into the branded icons while we're at it?

**Comment by francisrupert** [comment]
> Any other keywords you want to throw in...

That's a long list. We've done an _ok_ job of managing keywords, but definitely need to be more strict about it. At some point we'll retroactively add keywords in. Sounds fun right?

Good to go!

**Pull request merged**